migrations.lua

```diff
@@ -0,0 +1,16 @@
+local db = require("lapis.db")
+local schema = db.schema
+
+--[[
+    Make sure to use a date string YYYYMMDDHHMMSS as the migration key.
+]]
+
+return {
+    -- Initial setup.
```

Comment:
I'm removing this one, it's not used for anything and it will confuse people looking at the source code. Maybe it's better if you keep working on it on a separate branch in your fork?